app/org/sagebionetworks/bridge/dynamodb/DynamoSurveyDao.java

```diff
@@ -359,28 +363,40 @@ public Survey getSurveyMostRecentlyPublishedVersion(StudyIdentifier studyIdentif
         return new QueryBuilder().setStudy(studyIdentifier).isPublished().setSurvey(guid).isNotDeleted().getOne(true);
     }
     
-    // SCAN
-    @Override
-    public Survey getSurveyMostRecentlyPublishedVersionByIdentifier(StudyIdentifier studyIdentifier, String identifier) {
-        return new QueryBuilder().setStudy(studyIdentifier).setIdentifier(identifier).isPublished().isNotDeleted().getOne(true);
-    }
```

Comment:
I'm sure. In a UI, we'll start by showing a lit of the most recent surveys, and drill down from there. The SDK can convert these into a map by identifier, if necessary.